index.html

```diff
@@ -113,7 +113,8 @@ <h2>How can you use GitGitGadget?</h2>
 				You might also want to read the <a href="https://git-scm.com/docs/gitworkflows">gitworkflows</a> manual page to understand how your contributions will be integrated in Git's repository, as well as <a href="https://github.com/git/git/blob/todo/MaintNotes">this note from Git's maintainer</a>.
 			</p><p>
 				The first time you use GitGitGadget, you need to be added to the list of users with permission to use GitGitGadget (this is a <i>very</i> simple anti-spam measure).
-				Any user who is already on that list can do that, by adding a comment to that Pull Request that says <code>/allow &lt;username&gt;</code> (with your GitHub login name).
+				Using <a src="https://web.libera.chat/#git-devel">IRC</a> or <a src="https://github.com/gitgitgadget/git/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen+%22%2Fallow%22">PRs</a> you can find a user already on this list and request them to add you to the list.
+				To do so, they simply have to add a comment to that Pull Request that says <code>/allow</name></code> (with your GitHub login name).
```

Comment:
I would prefer to simply drop the "(with your GitHub login name)" part. It is not necessary and therefore only confusing the readers.

Otherwise it looks good!